### ISSUE TITLE
Search: give option to restrict search results

### ISSUE THREAD

**Issue by fresatomica** [Automattic/wp-calypso#58705]
### What

A user suggested a possibility of restricting search results to, for example, page titles.

### Why

> Example: I have a 300-page site with one page called “Sitemap” and links to that page in the content/menus on the other 299. If I do a search in the back end (Pages > All Pages) for “Sitemap”, I get 300 results, and the actual Sitemap page is way down the list, making it very difficult to find. All i need to find is that one page.
> 
> This applies to all pages on the site: if a search term (i.e. a page title) appears anywhere on a page – as a link, for example, or in a menu – the page appears in search results. I need to make this stop. 

Link to the request [here](https://wordpress.com/forums/topic/restrict-back-end-search-to-page-titles/).

**Comment by Greatdane** [comment]
I agree that the search function could do with some love. For example, even if I search 'home' on my site, the page titled 'home' does not show up first. I am not sure which logic is used for searching.
